trio/_tests/test_contextvars.py

```diff
@@ -18,7 +22,7 @@ async def child():
     assert record == ["main"]
 
 
-async def test_contextvars_set():
+async def test_contextvars_set() -> None:
```

Comment:
Probably not super necessary, but internal function `child` is missing it's annotation as well.